rpcs3/Emu/Cell/Modules/cellAudio.cpp

```diff
@@ -679,6 +679,7 @@ error_code cellAudioSetPortLevel(u32 portNum, float level)
 	cellAudio.trace("cellAudioSetPortLevel(portNum=%d, level=%f)", portNum, level);
 
 	const auto g_audio = g_idm->lock<named_thread<audio_thread>>(0);
+	level *= g_cfg.audio.volume / 100.0f;
```

Comment:
Shouldn't you move this to line 700?
Right now line will be redundant in case any of those other checks fail.

Reply:
Doesn't really make a huge difference, but I guess it's a more logical location.